lib/src/client_context.dart

```diff
@@ -24,18 +24,17 @@ abstract class ClientContext {
   Assets get assets;
 }
 
-abstract class Services {
-  DatastoreDB get db;
-
-  /// Access the gcloud package storage API. This will autumatically be
-  /// available on deployed App Engine applications. For local testing this
-  /// will be `null` unless the environemnt variable
-  /// `STORAGE_SERVICE_ACCOUNT_FILE` is passed in `app.yaml`. This environment
-  /// variable should point to a private key for a service account in JSON
-  /// format.
-  Storage get storage;
-  Logging get logging;
-  Memcache get memcache;
+class Services {
+  final DatastoreDB db;
+
+  /// Access the gcloud package storage API.
+  ///
+  /// This will automatically be available on deployed App Engine applications.
```

Comment:
This is not just for deployed applications. We require GCLOUD_KEY be available for local testing, so we are also guaranteed to have the [storageService] be available locally.

Just remove the comment!